packages/mobile/src/screens/profile-screen/ProfileHeader/ProfileSocials.tsx

```diff
@@ -62,13 +61,25 @@ export const ProfileSocials = () => {
     [tiktok_handle, TikTokSocialLink]
   ]
 
+  const opacity = useRef(new Animated.Value(0)).current
```

Comment:
general comment that in some places we fade from some nonzero start value to 1 which makes the animation feel nicer sometimes. we should standardize this but worth a tiny bit of digging (e.g. how track images fade in)